# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/security/test/https-only/browser.ini
# Commit: 9d14d5ae3863
# Full Hash: 9d14d5ae386341911507881b91c4e709795a9395
# Author: lyavor <lyavor@mozilla.com>
# Date: 2021-04-26 15:46:32
# Regressor Bug: 1683015
# File Overlap Count: 1
# Description:
#   Bug 1683015 HTTPS-Only Mode Alert appears on site which supports https. r=ckerschb,JulianWels
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112702
# ==============================================================================

diff -r 87457b540267 -r 9d14d5ae3863 dom/security/test/https-only/browser.ini
--- a/dom/security/test/https-only/browser.ini	Mon Apr 26 11:37:49 2021 +0000
+++ b/dom/security/test/https-only/browser.ini	Mon Apr 26 11:45:20 2021 +0000
@@ -10,3 +10,6 @@
 support-files =
   file_iframe_test.sjs
 [browser_triggering_principal_exemption.js]
+[browser_background_redirect.js]
+support-files =
+  file_background_redirect.sjs